blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -40,6 +42,12 @@ ALTER TABLE can be used to alter an existing table.
 While columns can be added at any time, adding a new
 :ref:`generated column <ref-generated-columns>` is only possible if the table is empty.
 
+``OPEN`` and ``CLOSE`` can be used to open or close the table, respectively.
+Closing a table prevents all operations, except ``ALTER TABLE ... OPEN``, to
+fail. Operations targetting a single closed partition will fail (except ``OPEN``),
```

Comment:
The behaviour has been changed to be consistent with all other partitioned table scenarios: if you did `where part_col=1`, it now just returns an empty result, similar to if you only had 1 partition, closed it, and then did `select * from table;`.